pkgs/by-name/li/linien-server/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, linien-common
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
./result/bin/linien-server 

```
  File "/nix/store/38aa8gm3nyssciinn10x9ym6c5gmmw4c-linien-server-0.8.0/lib/python3.11/site-packages/linien_server/registers.py", line 52, in __init__
    from linien_server.acquisition import AcquisitionService
  File "/nix/store/38aa8gm3nyssciinn10x9ym6c5gmmw4c-linien-server-0.8.0/lib/python3.11/site-packages/linien_server/acquisition.py", line 32, in <module>
    from pyrp3.board import RedPitaya  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'pyrp3'
Saved parameters to  /home/kirillvr/.local/share/linien/linien_parameters.json
```

Reply:
Should I package `pyrp3` too?

Reply:
If the software has a hard dependency on it, then probably yes

Reply:
Running now with:
```
./result/bin/linien-server --fake
```
https://github.com/linien-org/linien/wiki/Development#optional-install-fake-server